heidegger_index/models.py

```diff
@@ -125,6 +130,48 @@ def __str__(self):
         else:
             return f"{self.start}"
 
+    def refers_to_page(self, page):
+        try:
+            page = int(page)
+        except TypeError:
+            raise TypeError
+
+        if self.end:
+            if page >= self.start and page <= self.end:
+                return True
+        if not self.end and self.start:
+            if page == self.start:
+                return True
+            elif self.suffix == "f" and page == self.start + 1:
+                return True
+            elif self.suffix == "ff" and page == self.start + 2:
+                return True
+        else:
+            return False
```

Comment:
Als het eerste if-statement `True` is, maar de drie if-statements die daarin staan niet, dan wordt er geen `False` teruggegeven, maar `None`. Gaat in de praktijk wel goed denk ik, maar netter is om te zorgen dat `False` terugkomt. Mooie manier om dat te doen is door de laatste `else` weg te laten:

```suggestion
        if not self.end and self.start:
            if page == self.start:
                return True
            elif self.suffix == "f" and page == self.start + 1:
                return True
            elif self.suffix == "ff" and page == self.start + 2:
                return True

        return False
```